builder.json

```diff
@@ -14,6 +14,6 @@
     "downstream": [
     ],
     "test_steps": [
-        "test"
+        "aws-c-auth-test"
```

Comment:
Would have thought no originally but since env is a builder thing, it seems likely that steps that side-affect it could hold for subsequent steps so I'll give it a try.

Reply:
Tried this, but couldn't get it to work.  No idea why, as soon as I switched to two steps and renamed the steps (both here and in the python file) it just tried to run aws-c-auth-test-prep on the command line.

Reply:
Tried it again after it seemed to work on aws-crt-cpp and voila now it works.  Must have done something dumb the first time around.